Memeolist/Memeolist/view/comments/CommentsViewController.swift

```diff
@@ -0,0 +1,47 @@
+import AGSSync
+import UIKit
+
+class CommentsViewController: UIViewController {
+    
+    var meme: AllMemesQuery.Data.AllMeme?
+    var tableViewController: CommentsTableViewController?
```

Comment:
Any reason why this UIViewController is needed? Did you follow some specific tutorial? 
I'm more curious rather pointing that as an issue.